src/configuration/Configuration.cxx

```diff
@@ -16,6 +16,7 @@
 
 #include "configuration/Configuration.h"
 #include "configuration/HTTPClientConfiguration.h"
+#include "core/util/URLEncoding.h"
```

Comment:
I've noticed the different classes between the Native and the Java SDK. In Java SDK the class is called "PercentEncoder". Since w already have "URLEncoding" I think it OK to have the functionality in different classed.